tests/unit/test_transfer_client.py

```diff
@@ -1722,3 +1723,139 @@ def test_endpoint_manager_task_successful_transfers(self):
+                                            GO_EP1_ID, notify_on_fail=False)
+            tdata.add_item("/", "/", recursive=True)
+            task_ids.append(self.tc2.submit_transfer(tdata)["task_id"])
+
+        # track assets for cleanup
+        self.asset_cleanup.append(
+            {"function": self.tc.endpoint_manager_cancel_tasks,
+             "args": [task_ids, "Cleanup for unauthorized_transfers helper"]})
+
+        return task_ids
+
+    def test_endpoint_manager_cancel_tasks(self):
+        """
+        Get task ids from _unauthorized transfers, and has sdktester1a cancel
+        those tasks. Validates results.
+        Confirms 403 when non manager attempts to use this resource.
+        """
+        # cancel the tasks
+        task_ids = self._unauthorized_transfers()
+        message = "SDK test cancel tasks"
+        cancel_doc = self.tc.endpoint_manager_cancel_tasks(task_ids, message)
+
+        # validate results
+        self.assertEqual(cancel_doc["DATA_TYPE"], "admin_cancel")
+        self.assertIn("done", cancel_doc)
+        self.assertIn("id", cancel_doc)
+
+        # 403 for non managers, even if they submitted the tasks
+        with self.assertRaises(TransferAPIError) as apiErr:
+            self.tc2.endpoint_manager_cancel_tasks(task_ids, message)
+        self.assertEqual(apiErr.exception.http_status, 403)
+        self.assertEqual(apiErr.exception.code, "PermissionDenied")
+
+    def test_endpoint_manager_cancel_status(self):
+        """
+        Has sdktester2b submit three unauthorized transfers from the managed
+        endpoint, and sdktester1a admin_cancel those tasks.
+        Gets the cancel status of the cancel and validates results.
+        Loops while status is not done, then confirms all tasks canceled.
+        """
+        # cancel the tasks and get the cancel id
+        task_ids = self._unauthorized_transfers()
+        message = "SDK test cancel status"
+        cancel_id = self.tc.endpoint_manager_cancel_tasks(
+            task_ids, message)["id"]
+
+        # loop while not done or fail after 30 tries, 1 try per second
+        for tries in range(30):
+            # get and validate cancel status
+            status_doc = self.tc.endpoint_manager_cancel_status(cancel_id)
+            self.assertEqual(status_doc["DATA_TYPE"], "admin_cancel")
+            self.assertEqual(status_doc["id"], cancel_id)
+            if status_doc["done"]:
+                break
+            else:
+                time.sleep(1)
+
+        # confirm sdktester2b now sees all tasks as canceled by admin.
+        for task_id in task_ids:
+            task_doc = self.tc2.get_task(task_id)
+            self.assertEqual(task_doc["canceled_by_admin"], "SOURCE")
+            self.assertEqual(task_doc["canceled_by_admin_message"], message)
+
+    # TODO: uncomment these tests when
+    # https://github.com/globusonline/koa/issues/49
+    # is resolved.
+    '''
```

Comment:
Rather than commenting this out, can we use nose2's skip functionality? That still requires that we re-enable the tests, but we'll have better visibility/reminders because we'll see the tests being skipped in nose2 output.